### PR TITLE
Colw/api-validators-transactions

### PR DESCRIPTION
**Pull request by colw**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/network/NetworkItem.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
     :class="{ active: connection.network === network.id }"
   >
     <div class="network-icon">
-      <img :src="network.logo_url" alt="cosmic atom token" />
+      <img src="https://s3.amazonaws.com/network.logos/cosmos-logo.png" alt="cosmic atom token" />
```
Comment (faboweb) [inline src/components/network/NetworkItem.vue]: this is wrong. the logos are stored locally in Lunie not in a remote repo

Comment (colw) [reply src/components/network/NetworkItem.vue]: Of course, will update.

**Comment by faboweb** [comment]
Validators page loads very slow. I guess due to removal of the infinite scroll.

**Comment by colw** [comment]
> Validators page loads very slow. I guess due to removal of the infinite scroll.

I'll reinstate infinite scroll soon. It's a combination of that and the API requests on the backend.

File: src/components/network/NetworkItem.vue (hunk 1 above)
Comment (colw) [reply src/components/network/NetworkItem.vue]: 👀

**Pull request merged**